libkbfs/bcache_test.go

```diff
@@ -7,6 +7,8 @@ package libkbfs
 import (
 	"testing"
 
+	"golang.org/x/net/context"
```

Comment:
Does this need its own import block or can it be combined with the imports below as in most other files?